core/assets/bundles/bundle_nl.properties

```diff
@@ -928,12 +983,13 @@ block.crawler-factory.name = Crawler Mech Factory
 block.titan-factory.name = Titan Mech Factory
 block.fortress-factory.name = Fortress Mech Factory
 block.revenant-factory.name = Revenant Fighter Factory
-block.repair-point.name = Repair Point
+block.repair-point.name = APK Punt
 block.pulse-conduit.name = Pulse Conduit
+block.plated-conduit.name = Plated Conduit
```

Comment:
Gepantserde pijp

Reply:
Note that i didn't "add" this entry, it as a fresh copy from the english bundle